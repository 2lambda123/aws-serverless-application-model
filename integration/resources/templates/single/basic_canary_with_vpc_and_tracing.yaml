Resources:
  CanaryWithVPCAndTracing:
    Type: AWS::Serverless::Canary
    Properties:
      Handler: pageLoadBlueprint.handler
      InlineCode: |
        const loadBlueprint = async function () {
            console.log("hello")
        };
        exports.handler = async () => {
            return await loadBlueprint();
        };
      Runtime: syn-nodejs-puppeteer-3.1
      Schedule:
        Expression: rate(0 minutes)
      StartCanaryAfterCreation: true
      VpcConfig:
        SecurityGroupIds: []
        SubnetIds: []
      ActiveTracing: true