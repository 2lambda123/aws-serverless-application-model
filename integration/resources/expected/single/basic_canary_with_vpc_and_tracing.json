[
  { "LogicalResourceId":"CanaryWithVPCAndTracing", "ResourceType":"AWS::Synthetics::Canary" },
  { "LogicalResourceId":"CanaryWithVPCAndTracingRole", "ResourceType":"AWS::IAM::Role" },
  { "LogicalResourceId":"CanaryWithVPCAndTracingArtifactBucket", "ResourceType":"AWS::S3::Bucket" }
]